registry/emailTemplates.xml

```diff
@@ -69,4 +69,5 @@
 	<email key="REVISED_VERSION_NOTIFY" can_disable="0" to_role_id="16" stage_id="3" subject="emails.revisedVersionNotify.subject" description="emails.revisedVersionNotify.description" body="emails.revisedVersionNotify.body"/>
 	<email key="STATISTICS_REPORT_NOTIFICATION" can_disable="1" from_role_id="16" to_role_id="17" subject="emails.statisticsReportNotification.subject" description="emails.statisticsReportNotification.description" body="emails.statisticsReportNotification.body"/>
 	<email key="ANNOUNCEMENT" can_disable="0" from_role_id="16" to_role_id="1048576" subject="emails.announcement.subject" description="emails.announcement.description" body="emails.announcement.body"/>
+	<email key="EDITORIAL_REMINDER" can_disable="0" from_role_id="16" to_role_id="1048576" subject="emails.editorialReminder.subject" description="emails.announcement.description" body="emails.editorialReminder.body"/>
```

Comment:
We decided to move these properties except `subject` and `body` to the correspondent Mailable